docs/source/quantization.rst

```diff
@@ -190,10 +190,10 @@ Layers for the quantization-aware training
     * :class:`~torch.quantization.Observer` — Abstract base class for observers
 * Quantization configurations
     * :class:`~torch.quantization.QConfig` — Quantization configuration class
-    * ``default_qconfig`` — Same as ``QConfig(activation=default_observer, weight=default_weight_observer)`` (See :class:`~torch.quantization.QConfig.QConfig`)
-    * :attr:`~torch.quantization.default_qat_qconfig` — Same as ``QConfig(activation=default_fake_quant, weight=default_weight_fake_quant)`` (See :class:`~torch.quantization.QConfig.QConfig`)
-    * :attr:`~torch.quantization.default_dynamic_qconfig` — Same as ``QConfigDynamic(weight=default_weight_observer)`` (See :class:`~torch.quantization.QConfig.QConfigDynamic`)
-    * :attr:`~torch.quantization.float16_dynamic_qconfig` — Same as ``QConfigDynamic(weight=NoopObserver.with_args(dtype=torch.float16))`` (See :class:`~torch.quantization.QConfig.QConfigDynamic`)
+    * :attr:`~torch.quantization.default_qconfig` — Same as ``QConfig(activation=default_observer, weight=default_weight_observer)`` (See :class:`~torch.quantization.qconfig.QConfig`)
```

Comment:
`default_qconfig` doesn't exist as an attribute, so it is better to label it as ``default_qconfig``
![image](https://user-images.githubusercontent.com/5652049/66953340-5235de00-f02c-11e9-92c5-6719ccca7ff3.png)

Reply:
I will send a PR to set it as just verbatim text.